fireapp/lib/data/client/shift_request_client.dart

```diff
@@ -1,14 +1,26 @@
+import 'dart:convert';
+
 import 'package:fireapp/data/client/api/rest_client.dart';
 import 'package:fireapp/domain/models/shift_request.dart';
+import 'package:flutter/services.dart';
 import 'package:injectable/injectable.dart';
 
 @injectable
 class ShiftRequestClient {
   final RestClient restClient;
   ShiftRequestClient(this.restClient);
 
-  Future<List<ShiftRequest>> getShiftRequestsByRequestID(String requestID) {
-    return restClient.getShiftRequest(requestID);
+  Future<Map<String, dynamic>> _loadMockData() async {
```

Comment:
This is how I told you to do it, this is just a note for future reference. A better way to do this is to have an IShiftRequestClient interface and both a mock and real implementation. Then you can just inject the one you want.

This is fine though.